docs/general/run-a-node/prerequisites/hardware-recommendations.md

```diff
@@ -1,218 +1,195 @@
 # Hardware Requirements
 
-The Oasis Network is composed of multiple classes of nodes that participate in
-different committees.
+The Oasis Network is composed of multiple classes of nodes and services, such
+as:
+
+* Consensus Validator/Non-validator Node
```

Comment:
For consistency I used capitalized "Consensus" when it appear in the beginning of all bullet points.

Suggestion: It would have been enough to suggest these changes once and tell to apply it throughout the document.